### PR TITLE
sync 3.4 branch with master.

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hajapy** [comment]
Bump build number?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,6 +43,10 @@ build:
   number: 3
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
  number: 4
```

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
I guess we should build this for qt 5.9 as well, but it seems that 5.12 is almost ready to go

**Pull request merged**